### ISSUE TITLE
Reference to "/tmp/conda_envs/258853ccd52b78b326300497/" in pre-built release on pypi.org

### ISSUE THREAD

**Issue by kaivogelgesang** [Calysto/matlab_kernel#160]
The pre-built release on [pypi](https://pypi.org/project/matlab-kernel/#files) (matlab_kernel-0.17.0-py3-none-any.whl) references `/tmp/conda_envs/258853ccd52b78b326300497/bin/python` as python executable in `matlab_kernel/matlab/kernel.json`.
I suspect this happens because of [line 27 in setup.py](https://github.com/Calysto/matlab_kernel/blob/433723db67c75b4c3da0e08166827578a3e8bccc/setup.py#L27) when building this release.

This causes the following kernel error ([full traceback](https://github.com/Calysto/matlab_kernel/files/8643111/full_error.txt)) when opening Matlab notebooks:
```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/conda_envs/258853ccd52b78b326300497/bin/python'
```
since the conda environment is of course not present on my machine, rendering jupyter completely unusable.

# Steps to reproduce:
This was done on a machine running Pop!_OS 21.10 with Python 3.9.7 and Matlab R2022a

1) Create a new blank virtual environment
```
$ python3 -m venv .venv
```
2) Install matlab engine as documented [here](https://www.mathworks.com/help/matlab/matlab_external/install-the-matlab-engine-for-python.html)
```
$ sudo -s
# source .venv/bin/activate
# cd path/to/matlab/extern/engines/python
# python setup.py install
# exit
```
3) Install jupyter and matlab-kernel
```
$ source .venv/bin/activate
$ pip install jupyter matlab-kernel
```
(pip downloads the pre-built version from pypi, specifically [this file](https://files.pythonhosted.org/packages/ce/5e/475317c91ae0a94f7ed5abfeb77a2b7b23151e2f5f5f530c5f962a252a67/matlab_kernel-0.17.0-py3-none-any.whl))

4) Start `jupyter notebook` and create a new matlab notebook
5) Kernel error

**Comment by blink1073** [comment]
Thanks, good catch!  The way we handle this in `ipykernel` is to check for command line [arguments](https://github.com/ipython/ipykernel/blob/9bea8c588a25d7da8e421930840f2d015acf6983/setup.py#L108).

@kaivogelgesang, @RibomBalt are either of you available to make a PR with a fix?

**Comment by blink1073** [comment]
Fixed in 0.17.1, thanks again @RibomBalt!
